src/Microsoft.Dnx.Tooling/NuGet/ContentModel/Infrastructure/Parser.cs

```diff
@@ -110,7 +110,7 @@ internal override bool TryMatch(ContentItem item, IDictionary<string, ContentPro
                 ContentPropertyDefinition propertyDefinition;
```

Comment:
It's because they are copied from NuGet. I think it should be OK to add them /cc @Eilon